tests/cft/test_s3bench_workload.py

```diff
@@ -66,14 +67,26 @@ def test_19658(self):
                                        num_clients=clients, num_sample=samples,
                                        obj_name_pref="loadgen_test_", obj_size=size,
                                        skip_cleanup=False, duration=None,
-                                       log_file_prefix="TEST-19658", end_point=S3_CFG["s3_url"],
+                                       log_file_prefix=log_file_prefix, end_point=S3_CFG["s3_url"],
                                        validate_certs=S3_CFG["validate_certs"])
-                self.log.info(
-                    f"Loop: {loop} Workload: {samples} objects of {size} with {clients} parallel "
-                    f"clients.")
-                self.log.info(f"Log Path {resp[1]}")
+                self.log.info("Loop: %s Workload: %s objects of %s with %s parallel clients.",
+                              loop, samples, size, clients)
+                self.log.info("Log Path %s", resp[1])
                 assert not s3bench.check_log_file_error(resp[1]), \
-                    f"S3bench workload for failed in loop {loop}. Please read log file {resp[1]}"
+                    f"S3bench workload failed in loop {loop}. Please read log file {resp[1]}"
+
+    @pytest.mark.longevity
+    @pytest.mark.tags("TEST-19658")
+    def test_19658(self):
+        """Longevity Test with distributed workload."""
+        self.execute_workload_distribution("test_19658", "TEST-19658")
+
+    @pytest.mark.lc
+    @pytest.mark.s3_data_path
+    @pytest.mark.tags("TEST-39216")
+    def test_39216(self):
```

Comment:
can we name it as test_workload or something?

Reply:
Done